### DISCUSSION TITLE
Can `setPixelTolerance` vary with latitude?

### DISCUSSION THREAD

**Discussion by wipfli** [onthegomap/planetiler#558] [Q&A] [answered]
Is it possible to vary `setPixelTolerance` depending on the latitude of a tile?

![image](https://user-images.githubusercontent.com/53421382/233680444-6de688b9-28df-446b-9d56-ee1b5ee5e97d.png)

I have a situation where I want to simplify hexagonal H3 cells. In Alaska, they don't get simplified much and the hexagons are visible but further down they get simplified nicely.

https://wipfli.github.io/swiss-map/#map=2.69/56.36/-85.81

**Comment by wipfli** [comment]
If it is not a big problem if it is not possible to change `setPixelTolerance` with latitude. I find the hexagons actually look quite fun...

**Reply by msbarry** [reply, marked as answer]
Set pixel tolerance should always generate a consistent smoothing when rendered to web mercator since planetiler works in screen pixel coordinates. I think what's going on is the h3 cells get bigger at higher latitudes. I wonder if there's a variant that generates consistent sized cells in web Mercator?

If you really wanted to do this you could get the centroid of the feature when processing it, compute latitude and set the pixel tolerance based off of that
